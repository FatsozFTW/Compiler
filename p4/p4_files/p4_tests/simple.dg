bool glob;
int test;
int main(){
	int a;
    a = 1+3;
    test = test;
	glob = a;
}
