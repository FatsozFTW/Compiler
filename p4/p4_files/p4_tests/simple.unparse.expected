bool glob(bool);
int main(->int)(){
    int a(int);
    glob(bool) = a(int);
}
